clang/lib/CodeGen/CodeGenModule.cpp

```diff
@@ -6324,10 +6324,20 @@ void CodeGenModule::EmitGlobalFunctionDefinition(GlobalDecl GD,
 
   SetLLVMFunctionAttributesForDefinition(D, Fn);
 
+  auto getPriority = [this](auto *Attr) -> int {
+    int priority = Attr->DefaultPriority;
```

Comment:
```suggestion
    unsigned priority = Attr->DefaultPriority;
```
You made this unsigned earlier :)